chainercv/evaluations/eval_semantic_segmentation.py

```diff
@@ -77,30 +86,31 @@ class labels as values, which is obtained from inference.
         the number of images in the input.
```

Comment:
Or we can simply take only a list of arrays. I think this is not inconvenient because we can easily convert an array to a list of an array.